tools/mtouch/Assembly.cs

```diff
@@ -72,14 +72,22 @@ HashSet<string> ComputeDependencies (List<Exception> warnings)
 		}
 
 		// returns false if the assembly was not copied (because it was already up-to-date).
-		public bool CopyAssembly (string source, string target, bool copy_mdb = true)
+		public bool CopyAssembly (string source, string target, bool copy_mdb = true, bool strip = false)
```

Comment:
If I understand correctly, you're saying that we might end up duplicating assemblies that are only identical after stripping (but not before)?

If so, I think you're right, but I'm not sure how often that would be the case (Xamarin.iOS.dll would still be different after stripping, but mscorlib.dll might not be).

In any case I don't think this PR changes the current behavior.